packages/xo-web/src/common/intl/messages.js

```diff
@@ -1293,6 +1293,8 @@ const messages = {
     'Only the files of Delta Backup which are not on a SMB remote can be restored',
   remoteEnabled: 'Enabled',
   remoteError: 'Error',
+  remoteErrorMessage:
+    'This remote has an invalid URL ({url}). Please click on this button to keep this URL params or edit them!',
```

Comment:
{url}: URL invalid (colon in path). Click this button to change the URL to {newUrl}.